test/integration/envtest/shootretry/shootretry_suite_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shoot_test
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	logzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/envtest"
+	"github.com/gardener/gardener/test/framework"
```

Comment:
```suggestion
	"github.com/gardener/gardener/pkg/client/kubernetes"
	"github.com/gardener/gardener/pkg/envtest"
	"github.com/gardener/gardener/test/framework"

	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
	"k8s.io/client-go/rest"
	"sigs.k8s.io/controller-runtime/pkg/client"
	logf "sigs.k8s.io/controller-runtime/pkg/log"
	logzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
	"sigs.k8s.io/controller-runtime/pkg/manager"
```

Reply:
I'm not sure, why you are suggesting this.
The import blocks are well-formed (stdlib, external, local imports separated and sorted) and `goimports` (i.e. `make check`) accepts it. I thought, we agreed on just accepting such sorting, if it's accepted by `goimports`.

Should we formalize this agreement somewhere?
If you rather want to have stricter sorting than what `goimports` provides, let's forge that into some tooling that can automate sorting + verification. I would vote against it, because imports should generally be nothing to care about and we wanted to go with community-supported tools.

Reply:
Actually, I think we have some agreement on this but I do not recall if we have it somewhere documented. 
The policy was first block is always the built-in packages, then the second block is about the repository own packages and the third block is the 3rd party packages.

Reply:
For the sake of getting this PR in and not blocking on this discussion, I sorted the imports as suggested by you.